packages/salesforcedx-vscode-lwc/src/testSupport/index.ts

```diff
@@ -12,8 +13,26 @@ import { lwcTestIndexer } from './testIndexer';
 import { taskService } from './testRunner/taskService';
 import { testResultsWatcher } from './testRunner/testResultsWatcher';
 import { startWatchingEditorFocusChange } from './utils/context';
+import { workspaceService } from './workspace/workspaceService';
```

Comment:
```suggestion
import { workspaceService } from './workspace';
```
Since workspaceService is already exported by the module's index.ts